cumulus_library_data_metrics/manifest.toml

```diff
@@ -18,80 +19,80 @@ file_names = [
 
 [export_config]
 export_list = [
```

Comment:
Now is probably a good time to switch this to count_list/flat_list/meta_list arrays